locales/en/pn/message_open_warning.html

```diff
@@ -2,25 +2,20 @@
     <p>
 
         <body>
-            Se continui, il messaggio risulterà legalmente recapitato a te. Questo ti
-            permetterà di risparmiare tempo e denaro, evitando di ricevere una
-            raccomandata cartacea. Aprire il messaggio su IO equivale infatti a firmare
-            la ricevuta di ritorno di una raccomandata tradizionale.
+            Se continui, la notifica risulterà legalmente recapitata a te. Aprire il
```

Comment:
È corretto che sia in ita?

Reply:
Al momento si, PN è solo in italiano (anche il frontend web) e le entry in EN sono obbligatorie in app (anche se non tradotte).